src/main/java/com/example/HwLes11ANWM/dtos/UserDto.java

```diff
@@ -0,0 +1,65 @@
+package com.example.HwLes11ANWM.dtos;
+
+import com.example.HwLes11ANWM.models.Authority;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import java.util.Set;
+
+public class UserDto {
+
+    public String username;
+    public String password;
```

Comment:
Ik zou geen password in je dto zetten. Dat is gevoelige informatie die je niet wilt blootgeven. Dat geld ook voor een apikey. Voor authorities zou ik het ook zeggen, maar dat mag je wel teruggeven als je een goede reden hebt.